README.md

```diff
@@ -283,6 +283,8 @@ A callback executed after a search event. This is called once on initial render
 
 #### `copyEvent (=(e: KeyboardEvent) => e.key === "c" && (e.metaKey || e.ctrlKey))`
 
+#### `selectAllEvent (=(e: KeyboardEvent) => e.key === "a" && (e.metaKey || e.ctrlKey))`
```

Comment:
I think we'll want to move this to just below the existing description about `ctrl+c` and copy, and describe it's default keybinding `ctrl+a`

Reply:
fixed in 596d3ea51445930629df90b271a07c46ddcf1db7